### DISCUSSION TITLE
About the pronouns...

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#8219] [QA - tech support + general]
Curiosity dragged me and I went for it, even thoguh she knows I'm a guy.
She asked if I made some new discoveries and I said "Yes" and still a boy.
She said that it's the same as before, silly.

She took this one as a joke, right? I hope I don't get too worried over this...

**Comment by WiiBii2** [comment]
Hmm? I don't understand the problem... haha.
If you're worried about your choice leaving an impact, don't. MAS isn't an AI, it's just pre-coded scripts.

**Comment by Booplicate** [comment]
> If you're worried about your choice leaving an impact, don't. MAS isn't an AI, it's just pre-coded scripts.

That's not exactly how it works.

@hlepnamingblyat by her dialogue she's rather confused than thinking it's a joke. But you don't have to wait about it. It's more about her accepting you regardless of your gender.
